index.js

```diff
@@ -3,9 +3,9 @@ const childProcess = require('child_process');
 const execa = require('execa');
 const mem = require('mem');
 
-function getEnvVar() {
+function getEnvVar(options) {
```

Comment:
I don't see the point in an option, you can just always check `env.SUDO_USER` as it's only defined when sudo is used.